src/shared/utils/determineProgressColor.spec.ts

```diff
@@ -40,6 +40,7 @@ describe('determineProgressColor', () => {
   describe('coverage value is not a number', () => {
     it('returns default', () => {
       const result = determineProgressColor({
+        // @ts-expect-error
```

Comment:
what is this for?

Reply:
this was for the sake of testing a non-number val, I can swap to null though based off the other changes